src/main/java/edu/wpi/grip/ui/util/ImageConverter.java

```diff
@@ -34,12 +35,16 @@ public Image convert(Mat mat) {
         final int width = mat.cols();
         final int height = mat.rows();
         final int channels = mat.channels();
+        final int type = mat.type();
 
         assert channels == 3 || channels == 1 :
                 "Only 3-channel BGR images or single-channel grayscale images can be converted";
 
-        assert mat.type() == CV_8U || mat.type() == CV_8S :
-                "Only images with 8 bits per channel can be previewed";
+        // Note: If more channels are supported than this will need to be updated
+        assert Arrays.asList(
```

Comment:
So, I know streams are cool and all, but could you just rewrite this to use `||`?  So it's more readable.